bundles/model/org.eclipse.smarthome.model.rule.ide/src/org/eclipse/smarthome/model/rule/ide/RulesIdeModule.xtend

```diff
@@ -0,0 +1,11 @@
+/*
+ * generated by Xtext 2.12.0
+ */
+package org.eclipse.smarthome.model.rule.ide
+
+
+/**
+ * Use this class to register ide components.
+ */
+class RulesIdeModule extends AbstractRulesIdeModule {
```

Comment:
@sjka or anyone else, I cannot find this parent class when compiling locally (and GitHub can also not find it), I assume the smarthome build still works, so any hints?

Reply:
Keep the ctrl (or strg) key pressed and click on the `AbstractRulesIdeModule`.
It will direct you to that implementation.

Also a find for the respective file should give you the result:
```
$ find -name "AbstractRulesIdeModule.*"
./bundles/model/org.eclipse.smarthome.model.rule.ide/target/classes/org/eclipse/smarthome/model/rule/ide/AbstractRulesIdeModule.class
./bundles/model/org.eclipse.smarthome.model.rule.ide/src-gen/org/eclipse/smarthome/model/rule/ide/AbstractRulesIdeModule.java
```

Reply:
@maggu2810 thanks!

Then I fear that the generation failed somehow, because I had this error while compiling with everything with maven. It was while tracing a bug which disappeared anyway after a reboot...

Reply:
Hm, never (AFAIK) seen such an error while compiling using Maven. Are you sure that the Eclipse IDE has been closed while compiling with Maven?

Reply:
I'm sure it was not closed while compiling, I hear a possible work around / solution :-)

Reply:
You need to close the Eclipse IDE before you start a Maven build. :wink: